packages/utxo-lib/src/coinselect/inputs/branchAndBound.ts

```diff
@@ -134,7 +134,7 @@ export const branchAndBound: CoinSelectAlgorithm = (
     feeRate,
     options,
 ): CoinSelectResult => {
-    if (options.baseFee) return { fee: 0 }; // TEMP: disable bnb algorithm for DOGE
+    if (options.baseFee) return { fee: 0 }; // TEMP: disable bsc algorithm for DOGE
```

Comment:
I think in this case "bnb" refers to "branch and bound"

Reply:
🫣